src/components/OptionMenu/OptionMenu.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Link, useLocation } from "react-router-dom";
+import "./OptionMenu.css";
+export function OptionMenu() {
+  const location = useLocation();
+
+  return (
+    <div className="option-menu">
+      <Link to={"/notes"}>
+        <span
+          className={
+            location.pathname == "/notes"
+              ? "material-icons-outlined icon link active"
+              : "material-icons-outlined link "
+          }
```

Comment:
Better way of writing this would be:- 
`className={`material-icons-outlined link ${location.pathname === "/notes" ? "link active" ? ""}` }`

Reply:
Put back-ticks where text is gray.